i18n/zh.toml

```diff
@@ -215,7 +215,7 @@ other = "这是一个自动生成的文件，要修改其中的内容，请修
 other = "切换到英文版"
 
 [advisory]
-other = "中文内容由 ServiceMesher 社区维护，部分文档可能稍微滞后于英文版本，同步工作持续进行中"
+other = "中文内容由 Istio 社区中国开发者维护，部分文档可能稍微滞后于英文版本，同步工作持续进行中"
```

Comment:
```suggestion
other = "中文内容由 ServiceMesher 维护，部分内容可能稍微滞后于英文版本，同步工作持续进行中"
```

Reply:
First of all, [ServiceMesher](https://www.servicemesher.com) still exists and is a [part](https://github.com/istio/istio.io/pull/9460/files) of the [Cloud Native Community(China)](https://cloudnative.to). There have been three rounds of translation activities on the Istio website, all three of which were initiated by ServiceMesher, the latest round of which can be found in [https://github.com/istio/istio.io/issues/9128].  I propose to retain ServiceMesher's byline rights.

Reply:
We all thanks for your contributions from servicemesher community, i would suggest putting `servicemesher` to the contributor guide page(Contact with servicemesher can help quickly get involved). Meanwhile, anyone from anywhere can contribute to Chinese docs freely too., this is not limited to servicemesher. We should come to the consensus that all the code and docs belong to the istio contributors rather than another community.

@istio/steering-committee

Reply:
The main problem is that we should not have any developer advocacy group claim they are "official" if there isn't such endorsement from bylaw's perspective. Contributions are highly appreciated and valued however it is also necessary to be open to public, not to pick sides

Reply:
The translation of the Istio documentation was done entirely through volunteers, with the first two rounds organized through ServiceMesher and now by the Cloud Native Community Istio SIG, without the organizations in which such high quality and consistent translation work would not have been possible. I fully agree that the Istio Chinese documentation is copyrighted by the Istio community. 

I have a proposal:
- Add a document under [this link](https://istio.io/latest/zh/about/contribute/)(for example https://istio.io/latest/zh/about/contribute/zh-lang), "Get involved in Chinese translation", the document will contain the content in [this](https://i.cloudnative.to/istio/event/istio-doc-translation).
- Modify the phrase ”中文内容由 ServiceMesher 社区维护，部分文档可能稍微滞后于英文版本，同步工作持续进行中“ below the Chinese website to "部分内容可能滞后于英文版本，同步工作仍在进行中，[欢迎参与到中文翻译中来]()", which will redirect to the link above(https://istio.io/latest/zh/about/contribute/zh-lang).

I look forward to your reply. @istio/steering-committee

Reply:
I don't agree we should mark in official Istio doc that service mesher community maintains the Chinese doc. we don't want to require everyone to join the service mesher community to be able to contribute to istio Chinese doc.

Reply:
I agree that you don't need to join ServiceMesher to participate in contributing to the Istio Chinese documentation. But in fact, all three translations of the official documentation were done in the community I organized. We have accumulated a set of best practices. The text in the footer can be removed, but a page needs to be created on the Istio website to tell people how to participate in the Chinese translation.

Reply:
I've developed workflows and corresponding toolchains for Chinese translations, contributed hundreds of Commits, and numerous Reviews. I think I'm a contributor to the Istio community, not to other organizations.

Reply:
@fleeto You are a contributor to the Istio community, that's correct without doubt, so am I. You said you are not belong to other organizations, then please leave all the 'Wechat Group' and 'Github Organization' of ServiceMesher & Cloud Native Community. Many thanks :)

Reply:
I will vote for `Istio 社区中国开发者维护` -- Means all contributors from China.   

All  Istio contributors from China should be part of the whole Istio community.  This is also what an open source community operates- regardless to the country, location, we are a big  and Single Istio Community.

Reply:
@rootsongjc From the community view, we(@istio/steering-committee ) would not want `servicemesher` to occur, we have to claim there is only one community here. @istio/wg-docs-maintainers-chinese What's your opinion?

As to how to join Chinese translation, my own suggestion is to release tasks in Doc WG.

Reply:
It doesn't matter whether ServiceMesher is removed from the footer or not, the contributions and successes themselves belong to the Istio community. "Not want ServiceMaster to occur", does this phrase mean that it cannot appear on any page of Istio.io? Not even a word of acknowledgment, like it never existed? 

I thought ServiceMesher was a partner of the Istio community in China, and our work has made more people in China aware of Istio. See [this page](https://istio.io/latest/about/community/join/).

![image](https://user-images.githubusercontent.com/3328185/114834745-74681200-9e03-11eb-93f6-d66c0dbfeba6.png)

Is this phrase reserved? Or is it to be deleted as well?

On behalf of ServiceMesher, I just need an official acknowledgment from the Istio community, and we will continue to help maintain the Chinese documentation in the future.

Reply:
>Not even a word of acknowledgment, like it never existed?

We talked about this, and have not come to an agreement yet. From community view, every contributor should be thanked equally .

Reply:
IMO, a thank you page is reasonable.

Reply:
>Is this phrase reserved? Or is it to be deleted as well?
Good point, I think this should be removed as well.